### PR TITLE
Ignore SQLite system indices

### PR DESCRIPTION
**Pull request by aeneasr**
This resolves an issue where SQLite system indices were re-created when executing `change_column`, which throws an error in SQLite.

### CODE REVIEW

**Comment by aeneasr** [comment]
@stanislas-m sorry for pinging you here but you were the one that reviewed past request. I wasn't able to write tests for these 4-5 PRs yet but if this is wanted I can supply some for the individual PRs although I might fail with some internals, we'll have to see ;)

In any case, the PRs resolve many edge cases especially around SQLite and definitely improve the debugability.

**Comment by aeneasr** [comment]
For more context: The problem is that SQLite re-creates the table (renames the old ones and then copies the data) when using `alter_column`. The method that collects the table info also collects all the indices belonging to the table. This includes indices generated by SQLite for e.g. unique constraints. Because of that, the `alter_column` statement generates additional `CREATE INDEX` fields that create the indices that SQLite generates automatically. This causes an error because they are prefix with `sqlite_`, wich is a protected key. By updating the WHERE statement, these keys are ignored and only actual addicional keys (e.g. composite keys, regular indices, ...) are included.

File: translators/sqlite_meta.go [hunk 1]
```diff
@@ -111,7 +111,9 @@ func (p *sqliteSchema) buildTableData(table *fizz.Table, db *sql.DB) error {
 }
 
 func (p *sqliteSchema) buildTableIndexes(t *fizz.Table, db *sql.DB) error {
-	prag := fmt.Sprintf(`SELECT "seq", "name", "unique", "origin", "partial" FROM pragma_index_list('%s')`, t.Name)
+	// This ignores all internal SQLite keys which are prefixed with `sqlite_` as explained here:
+	// https://www.sqlite.org/fileformat2.html#intschema
+	prag := fmt.Sprintf(`SELECT "seq", "name", "unique", "origin", "partial" FROM pragma_index_list('%s') WHERE "name" NOT LIKE 'sqlite%%'`, t.Name)
```
Comment (stanislas-m) [inline translators/sqlite_meta.go]: Just a little detail: `SQLite prohibits applications from creating objects whose names begin with "sqlite_". ` So here, I guess it should be `WHERE "name" NOT LIKE 'sqlite_%%'` (`sqlitey` seems to be a valid prefix, if I read the docs correctly).

**Comment by stanislas-m** [comment]
Thanks!
